api/v1alpha4/awscluster_conversion.go

```diff
@@ -46,6 +49,37 @@ func (src *AWSCluster) ConvertTo(dstRaw conversion.Hub) error {
 	}
 
 	dst.Spec.S3Bucket = restored.Spec.S3Bucket
+	if restored.Spec.NetworkSpec.VPC.IPv6 != nil {
```

Comment:
Err... if I don't do the conversion the conversion fuzzy test fails that it can be set in the other thing but not in this thing. How do I fix that? Do I just nil this field, or... ?

Reply:
As the NetworkSpec struct is used by `AWSCluster` & `AWSManagedControlPlane` the fuzzing tests will try and populate it. 

There are 2 ways we handle this for `AWSCluster`

- support roundtrip conversation with `AWSCluster` and restore the values...even though the validating webhooks won't allow setting ipv6.
- Add a fuzzer function for populating `NetworkSpec` when its used by `AWSCluster` so it always nils the IPv6 part

I'd say go with the first option so that when we do support IPv6 with AWSCluster we don't have to go and remove fuzzing functions....plus the first option is already written.

Reply:
Yeah, that's largely what I understood as well. Also, the last point is a good point. We DO plan on supporting this in AWSCluster after I figure out why the heck the nodes aren't getting the right IP ( Oh did I mention that I figured out why unmanaged isn't working? It's because the nodes have ipv4 private addresses instead of ipv6 for some reason... ). :))